Fatal error: exception Failure("Attempt to reassign to "a" from closure")
